pkg/urbit/noun/nock.c

```diff
@@ -1700,6 +1702,14 @@ _n_hilt_fore(u3_noun hin, u3_noun bus, u3_noun* out)
     u3_atom now = u3i_chub(u3t_trace_time());
     *out = u3i_cell(u3h(hin), now);
   }
+  else if ( c3__nara == u3h(hin) ) {
+    u3t_slog_nara(0);
+    *out = u3_nul;
+  }
+  else if ( c3__hela == u3h(hin) ) {
+    u3t_slog_hela(0);
+    *out = u3_nul;
+  }
```

Comment:
With the two new hints here, I think it's worth converting this to a `switch` statement, as we did in `_n_hint_fore()`.